src/mappings/_utils.ts

```diff
@@ -125,3 +129,44 @@ export async function friendlyAmount(currency: Currency, amount: number): Promis
             return `Unknown asset: ${currency}`
     }
 }
+
+let currencyMap = new Map<CurrencyIdentifier, CurrencyExt>();
+
+export async function currencyToLibCurrencyExt(currency: Currency): Promise<CurrencyExt> {
```

Comment:
@peterslany : if I remember correctly, the lib will support more CurrencyExt types soon™ (AMM lp tokens, etc), is that correct?
If so, can you point me to the issues to add those - I would like to add squid issues to implement that after the lib work is merged.

Reply:
@bvotteler There was this issue https://github.com/interlay/interbtc-api/issues/531 to support LP tokens. So on squid side we need to add support for these too, but I was thinking you already done that, is that right? 

If so I think the only outstanding work will be to add lpToken and stableLpToken identifiers here:
https://github.com/interlay/interbtc-squid/blob/61a134d44510b60a25b3420e736221e9fd277d75/src/mappings/_utils.ts#L139-L150

using the lib implementation from here:
https://github.com/interlay/interbtc-api/blob/wip-feat-amm/src/utils/currency.ts#L186-L203

Reply:
I added these new types as comments to be included once those types are merged into the lib